### DISCUSSION TITLE
When will this mod be compatible with sodium?

### DISCUSSION THREAD

**Discussion by pityalvarez** [iPortalTeam/ImmersivePortalsMod#1262] [Q&A] [answered]
Specifically 1.16.5 becuase 1.16.4 is the same story. Sodium 0.2.0 and Inmersive portals 0.102, the last updates of both do not work.

**Comment by qouteall** [comment, marked as answer]
https://qouteall.fun/immptl/wiki/#sodium-compatibility

**Reply by FoxofGoodVibes** [reply]
1.21.1?? 
![image](https://github.com/user-attachments/assets/84e0f4ba-708c-4204-b4bb-364cc098e436)
